helper/testhelpers/docker/testhelpers.go

```diff
@@ -131,10 +131,21 @@ func (s ServiceURL) URL() *url.URL {
 // connection string (typically a URL) and nil, or empty string and an error.
 type ServiceAdapter func(ctx context.Context, host string, port int) (ServiceConfig, error)
 
+// This is now a wrapper function for StartNewService, passing the default values of
+// 'true' for addSuffix and 'false' for forceLocalAddr. Most tests can use this.
```

Comment:
Thanks for adding a comment here. I think it'd be better to focus the comment on what it does instead of how it does it (esp. for exported functions). As a new reader of these, the comment doesn't exactly make it obvious what happens when I call `StartService` or `StartNewService`. Thinking something along the lines of "StartService starts a new service/container with ...".